benchexec/tools/sbt-fizzer.py

```diff
@@ -0,0 +1,148 @@
+import benchexec.tools.template
+import benchexec.result as result
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import os
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for sbt-fizzer.
+    """
+
+    def name(self):
+        """
+        Return the name of the tool, formatted for humans.
+        This method always needs to be overriden, and typically just contains
+        return "My Toolname"
+        @return a non-empty string
+        """
+        return "sbt-fizzer"
+
+    def executable(self, tool_locator):
+        """
+        Find the path to the executable file that will get executed.
+        This method always needs to be overridden,
+        and should typically delegate to our utility method find_executable. Example:
+        return tool_locator.find_executable("mytool")
+        The path returned should be relative to the current directory.
+        @param tool_locator: an instance of class ToolLocator
+        @return a string pointing to an executable file
+        """
+        return tool_locator.find_executable("sbt-fizzer.py")
+
+    def version(self, executable):
+        """
+        Determine a version string for this tool, if available.
+        Do not hard-code a version in this function, either extract the version
+        from the tool or do not return a version at all.
+        There is a helper function `self._version_from_tool`
+        that should work with most tools, you only need to extract the version number
+        from the returned tool output.
+        @return a (possibly empty) string
+        """
+        return ""
```

Comment:
This method does not return the tool's version, which means that users will not see in the result tables which tool version was benchmarked. This can be confusing and hinders reproducibility.

The best would be to add a parameter like `--version` to your tool and read and return the version number from the method `version`. You can call the utility method `self._version_from_tool` to do this easily.